.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Why?
```

Comment:
Not sure if the template applies for playbook

Reply:
PR template is unrelated to this PR's title, so pls propose in different PR.